08/kth-missing-positive-number.md

```diff
@@ -86,7 +201,7 @@ Here are some example clarifying questions:
     * Consider at least one nominal and one edge case.
     * What is the expected output for the given input?
     * You can use the examples provided in the prompt, or other examples.
-2. Write unit tests for `kth_missing_positive_number` for the nominal and edge cases you identified in the first step.
+2. Write unit tests for `find_missing_positive_number` for the nominal and edge cases you identified in the first step.
```

Comment:
Regex replacement of "find_missing_positive_number" to "find_kth_missing_positive_number" worked fine in this file.